openedx/adg/common/djangoapps/user_authn_override/views/registration_form.py

```diff
@@ -0,0 +1,145 @@
+"""
+Overridden RegistrationFormFactory from openedx.core.djangoapps.user_authn.views.registration_form.py
+"""
+
+
+from django.utils.translation import ugettext as _
+
+from edxmako.shortcuts import marketing_link
+from openedx.adg.constants import ADG_BUSINESS_LINES, CITIES_OF_KSA
+from openedx.core.djangoapps.user_api import accounts
+from openedx.core.djangoapps.user_authn.views.registration_form import (
+    RegistrationFormFactory as CoreRegistrationFormFactory
+)
+from openedx.core.djangolib.markup import HTML, Text
+
+
+class RegistrationFormFactory(CoreRegistrationFormFactory):
+    """
+    Overridden openedx RegistrationFormFactory to avoid core changes.
+    """
+
+    EXTRA_FIELDS = CoreRegistrationFormFactory.EXTRA_FIELDS + ["is_adg_employee"]
+
+    def _add_username_field(self, form_desc, required=True):
+        """Add a username field to a form description.
```

Comment:
Can we add vertical spacing too for these docs ? 
Reference: 
https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html    
"""
    Summary line.

    Extended description of function.

    Arguments:
        arg1 (int): Description of arg1
        arg2 (str): Description of arg2

    Returns:
        bool: Description of return value
    """